packages/components/src/components/banner/Banner.tsx

```diff
@@ -44,7 +44,7 @@ export const Banner: React.FC<BannerProps> = ({
 
       {/* Close button (optional) */}
       {isClosable && onClose && (
-        <div className="tk-banner__close" onClick={onClose} />
+        <button className="tk-banner__close" onClick={onClose} />
```

Comment:
You would need to specify a `type="button"` here, as the default on eis `submit`.

Someone using the component inside a form would have it submitted when closing it.

Reply:
updated, thanks !